packages/synapse-react-client/src/mocks/msw/handlers/accessRequirementHandlers.ts

```diff
@@ -19,8 +20,13 @@ import { SynapseApiResponse } from '../handlers'
 import {
   mockAccessRequirements,
   mockAccessRequirementWikiPageKeys,
+  mockSelfSignAccessRequirement,
 } from '../../mockAccessRequirements'
 import { mockApprovedSubmission } from '../../dataaccess/MockSubmission'
+import { MOCK_USER_ID } from '../../user/mock_user_profile'
+
+const accessRequirementStatuses: Map<string, AccessRequirementStatus> =
+  new Map()
 
 export const getAccessRequirementHandlers = (backendOrigin: string) => [
```

Comment:
Store AR status in a map variable so we can mutate AR status on-the-fly (e.g. when we simulate accepting a click-wrap)